src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/dataaggregation/AreaController.java

```diff
@@ -62,6 +62,9 @@ public Response area(HttpServletRequest servletRequest, HttpServletResponse serv
    */
   @ApiOperation(value = "Area of OSM elements grouped by the type", nickname = "areaGroupByType",
       response = GroupByResponse.class)
+  @ApiImplicitParam(name = "filter", value = ParameterDescriptions.FILTER,
+      defaultValue = DefaultSwaggerParameters.BUILDING_FILTER, paramType = "query",
+      dataType = "string", required = false)
```

Comment:
Was the filter missing as default in the previous versions :thinking: Or was this inherited from another point in the code?

Reply:
all requests have a default filter set in [`SwaggerConfig.java`](https://github.com/GIScience/ohsome-api/pull/116/files#diff-0ca4bdeebb24a703111abee77d39050f88c0d4c9f79a8de4fda5172ff0433568R201-R204). But for the `/groupBy/type` requests I think that the generic filter (which only looks at `way` elements) doesn't make much sense, so I overwrite it with the `building=*` filter here.